modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js

```diff
@@ -28,7 +28,12 @@ const formatActions = (actions, itemData) => {
 	return actions
 		? actions.reduce((actions, action) => {
 				if (action.data?.permissionKey) {
-					if (itemData.actions?.[action.data.permissionKey]) {
```

Comment:
Maybe we could simplify the item actions use case, using only `action.data.permissionKey.toLowerCase()` as I cannot find an example of `camelCase` actions.

Reply:
It's probably better to convert, like in current solution. It's a bit more future-proof :+1: